parm/archive/gefs_wave.yaml.j2

```diff
@@ -0,0 +1,55 @@
+{% set cycle_HH = current_cycle | strftime("%H") %}
+{% set cycle_YMD = current_cycle | to_YMD %}
+{% set cycle_YMDH = current_cycle | to_YMDH %}
+{% set head = RUN + "wave.t" + cycle_HH + "z." %}
+
+gefs_wave:
+    name: "GEFS_WAVE"
+    target: "{{ ATARDIR }}/{{ cycle_YMDH }}/gefs_wave.tar"
+    required:
+
+#need some logic here to select FHOUT_WAV or FHOUT_HF_WAV
+
+
+{% if REPLAY_ICS == "YES" %}
+    {% set ofst_hr = OFFSET_START_HOUR %}
+{% else %}
+    {% set ofst_hr = FHMIN_GFS %}
+{% endif %}
+
+{% if CASE == "C48" %}
+    {% set res = '5p00' %}
+{% elif CASE == "C96" or "C192" %}
+    {% set res = '1p00' %}
+{% elif CASE == "C384" %}
+    {% set res = '0p25' %}
+{% elif CASE == "C768" %}
+    {% set res = '0p25' %}
+{% endif %}
```

Comment:
Similar to above, this should be based on the `wavres` the extract job is using.